modules/installation/proc-installing-rhdh-on-openshift-with-helm-cli.adoc

```diff
@@ -46,3 +46,5 @@ echo "https://redhat-developer-hub-$NAMESPACE.$CLUSTER_ROUTER_BASE"
 .Verification
 * Open the running {product-short} instance URL in your browser to use {product-short}.
 
+.Additional resources
+* For more information, see link:https://docs.openshift.com/container-platform/4.16/applications/working_with_helm_charts/installing-helm.html[Installing Helm] in the {ocp-short} documentation.
```

Comment:
curious why we didnt' just link to this from line 11:

```
* You have installed the link:https://docs.openshift.com/container-platform/4.16/applications/working_with_helm_charts/installing-helm.html[Helm tool].
```

Reply:
@nickboldt That is an option; I just followed the instructions in the Jira to add this as an Additional resource. I know there are some opinions in CCS that we should include all links in an Additional resources section and not in-line, but we don't follow that consistently in RHDH. 

@Preeticp is there a specific reason why you recommended adding this link in the Additional resources sections rather than in the Prerequisites section?

Reply:
Sorry for the late response, but we could link it to the prerequisite on line 11.

Reply:
could you open a new PR @Preeticp  ? this was merged 3 weeks ago.